app/grandchallenge/workstation_configs/models.py

```diff
@@ -257,7 +257,7 @@ class ImageInterpolationType(models.TextChoices):
     )
 
     class Meta(TitleSlugDescriptionModel.Meta, UUIDModel.Meta):
-        ordering = ("created", "creator")
+        ordering = ("title",)
```

Comment:
This requires a migration file to be generated and committed.

Reply:
But actually, IMO it's best not to change the default ordering and better to do this in the view instead.